code/game/objects/objs.dm

```diff
@@ -12,8 +12,10 @@
 	var/damtype = "brute"
 	var/armor_penetration = 0
 	var/corporation = null
+	var/heat = 0
 
-
+/obj/proc/is_hot()
+	return heat
```

Comment:
That really should be `/obj` level proc/var?

Reply:
All objects can collide with mobs. Therefor all objects should have such property.